src/main/resources/hudson/plugins/sshslaves/Messages.properties

```diff
@@ -10,7 +10,8 @@ SSHLauncher.ConnectionClosed={0} [SSH] Connection closed.
 SSHLauncher.ErrorWhileClosingConnection=Exception thrown while closing connection.
 SSHLauncher.AbortedDuringConnectionOpen=Slave start aborted.
 SSHLauncher.FailedToDetectEnvironment=Failed to detect the environment for automatic JDK installation. Please report this to jenkinsci-users@googlegroups.com: {0}
-SSHLauncher.NoJavaFound=Java version {0} was found but 1.5 or later is needed.
+SSHLauncher.NoJavaFound=Java version {0} was found but 1.7 or later is needed.
```

Comment:
> I cannot modify the Spanish file

Exactly my point: once you delete the English key, translators should be able to see that they have stale keys in their translations and delete them on their own.